OrbitLinuxTracing/PerfEventReaders.h

```diff
@@ -19,9 +19,10 @@
 
 namespace orbit_linux_tracing {
 
-// Helper functions for reads from a perf_event_open ring buffer that require
-// more complex operations than simply copying an entire perf_event_open record.
```

Comment:
This was not a comment on the function but on the whole file, do you want it removed?